### PR TITLE
Add area owners for System.Formats

### PR DESCRIPTION
**Pull request by carlossanlop**

### CODE REVIEW

**Review by bartonjs — APPROVED** [review]

**Comment by danmosemsft** [comment]
You may want to "retcon" these labels on a bunch of older issues (search for keyword, bulk edit) so the bot has a chance to learn.

**Comment by danmosemsft** [comment]
Would you mind also adding https://github.com/dotnet/runtime/labels/area-System.Net.Quic with the same owners as for System.Net.Http? It's another new label

File: docs/area-owners.md [hunk 1]
```diff
@@ -78,6 +78,8 @@ Note: Editing this file doesn't update the mapping used by the `@msftbot` issue
...
 | area-System.Drawing                            | @jeffhandley  | @safern @tannergooding                              |                                                                                                                                                                                                                                                                                           |
 | area-System.Dynamic.Runtime                    | @jaredpar     | @cston @333fred                                     | Archived component - limited churn/contributions (see [#27790](https://github.com/dotnet/runtime/issues/27790))                                                                                                                                                                            |
+| area-System.Formats.Asn1                       | @jeffhandley  | @bartonjs                                           |
+| area-System.Formats.Cbor                       | @jeffhandley  | @eiriktsarpalis                                     |
```
Comment (jeffhandley) [inline docs/area-owners.md:82]: @bartonjs -- can you be a secondary owner of Cbor?
@eiriktsarpalis and @GrabYourPitchforks -- can you be additional owners of Asn1?

Comment (bartonjs) [reply docs/area-owners.md:82]: OK by me

Comment (GrabYourPitchforks) [reply docs/area-owners.md:82]: Sure, I can be an ASN.1 alt.

Comment (eiriktsarpalis) [reply docs/area-owners.md:82]: I wouldn't mind being the primary owner either.

**Review by jeffhandley — APPROVED** [review]

**Comment by carlossanlop** [comment]
I did the retcon as suggested for some closed Cbor/Asn1 issues. Hopefully that'll help the bot.

**Pull request merged**